example/main.go

```diff
@@ -39,6 +37,8 @@ type Size interface {
 	Size() int64
 }
 
+var verbose bool
```

Comment:
We can probably remove the verbose flag altogether and add a reference to the QUIC_GO_LOG_LEVEL env.

Reply:
Since this not only applies to those examples but to all programs using this package, I added a comment to the README instead.